microsoftgraph/graph-powershell-1.0/Microsoft.Graph.Authentication/Add-MgEnvironment.md

```diff
@@ -0,0 +1,124 @@
+﻿---
+external help file: Microsoft.Graph.Authentication.dll-Help.xml
+Module Name: Microsoft.Graph.Authentication
+online version: https://learn.microsoft.com/en-us/powershell/module/microsoft.graph.authentication/add-mgenvironment
+schema: 2.0.0
+---
+
+# Add-MgEnvironment
+
+## SYNOPSIS
+{{ Fill in the Synopsis }}
```

Comment:
Some help files under `graph-powershell-1.0/Microsoft.Graph.Authentication` are missing `synopsis`, `description`, `examples`, and `parameter descriptions`. IMO, the new docs shouldn't ship until we have complete content to avoid backlash.